card/lib/card/act_manager.rb

```diff
@@ -107,9 +107,19 @@ def clear
         directors.each_pair do |card, _dir|
           card.director = nil
         end
+        expire
         @directors = nil
       end
 
+      def expire
+        expirees.each { |expiree| Card.expire expiree }
+        @expirees = []
```

Comment:
only expire at the very end of the act.

could also happen at the very end of storage, so long as it's after all the subcards have finished  the storage phase, too.